### ISSUE TITLE
import Image

### ISSUE THREAD

**Issue by fmehdi15** [manoharan-lab/holopy#29]
In the image_file_io.py, I had to change the line "import Image as PILImage" to "from PIL import Image as PILImage"

**Comment by anna-wang** [comment]
I think this depends on how Image is installed (in the python path vs as a subdirectory of PIL). In the current image_file_io.py file it already tries to take this into account with the following lines:
```
try:
    import PIL.Image as PILImage
except ImportError: # pragma: no cover
    try:
        import Image as PILImage
    except ImportError:
        import image as PILImage
```
If you open ipython and type "import PIL.Image as PILImage" what happens?

Anna

**Comment by fmehdi15** [comment]
Thanks Anna. I could not test it as I am travelling till but will get back
to you after the first week of September.

Best

Faraz

**Comment by barkls** [comment]
Make sure this is fixed when addressing #12

**Comment by barkls** [comment]
fixed in #75
